bcgov/citz-imb-build-with-us-1
# Added frontend auth system and token storage

# Summary


Setup frontend auth system

Added authService which provides the user with the user's access token and decoded access token.


## How to test

1. Login via the login button on the LandingPage
2. Verify that you are now able to view options in the header bar
3. Uncomment lines 20-22 in LandingPage.jsx and verify that you can see your accessToken and decoded access token (in the form of userData) in the console

[![](https://img.shields.io/badge/tickets_and_issues_this_solves-blue?style=for-the-badge)](https://hamzamohdzubair.github.io/redant/)



- None listed.

## Type of change


- [ ] Dependency updates (may include refactoring of a dependencies functionality).

## How Has This Been Tested?




## Checklist:


- [ ] I have read the checklist.
